solr/core/src/java/org/apache/solr/spelling/AbstractLuceneSpellChecker.java

```diff
@@ -81,7 +81,6 @@ public abstract class AbstractLuceneSpellChecker extends SolrSpellChecker {
   public String init(NamedList<?> config, SolrCore core) {
     super.init(config, core);
     indexDir = (String) config.get(INDEX_DIR);
-    String accuracy = (String) config.get(ACCURACY);
```

Comment:
I think this is probably best for another JIRA..    I like consistency across things where possible because it just removes those small paper cut feelings that users get when trying out the various spell checkers...

Reply:
Just to clarify - would you prefer to have this particular change about the `accuracy` type removed from this PR?

Reply:
I think it's fine!  thanks...